src/__tests__/pic-definition.spec.js

```diff
@@ -1,24 +1,26 @@
+import * as axis from "../components/axis";
+import * as cells from "../components/cells";
+import * as columns from "../components/columns";
+import * as disclaimer from "../components/disclaimer";
+import * as tooltip from "../components/tooltip";
+import picDefinition from "../pic-definition";
+import * as scales from "../scales";
+
 const mock = ({
```

Comment:
This way of mocking do seem a bit odd to me, but from a converting perspective I guess it make sense to just keep it simple by not changing to much